### PR TITLE
Add nys settings

### PR DESCRIPTION
**Pull request by segfault**
NYS specific settings as a migration to setup the database state. This uses a time based id to avoid future conflicts with upstream migration changes. This will need to be documented in the upstream repo.

### CODE REVIEW

File: migrations/202008131450.do.nys-settings.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
...ID-19 pandemic\\nresponse app, one that only performs contact tracing. The clear\\nalignment of purposes of the app functions with respect to the response\\nto the pandemic.\\n\\nThe detailed assessment of the app against the above guidelines are to\\nbe published online.[^8]\\n\\nIdentify and Assess Risks\\n-------------------------\\n\\nAppendix E sets out the risks that have been identified for the project\\nand the levels for those risks *if not mitigated*. Overall risk score\\nfor each risk identified is calculated as the product of the risk\\nlikelihood score and the risk impact score (i.e. likelihood score X\\nimpact score). The following sets out the metrics used in documenting\\nthe risk assessment.\\n\\nIdentify Measures to Reduce Risks\\n---------------------------------\\n\\nAn evaluation of the identified risks in the previous section has been\\ncarried out and a series of measures have been detailed that seek to\\nmitigate those risks to an acceptable level. The table in Appendix F\\nsets out these mitigation measures and an assessment of the risk impact\\ndue to their introduction. The table also sets out if these mitigation\\nmeasures have been approved.\\n\\n"}}');
```
Comment (jackmurdoch) [inline migrations/202008131450.do.nys-settings.sql]: You've got some un-escaped quotes in here which will cause problems. Beyond this, the general approach looks good to me (as long as these NYS specific values don't end up in the upstream repo).

Comment (dennisgove) [reply migrations/202008131450.do.nys-settings.sql]: The first INSERT statement fails with error 
```
ERROR:  syntax error at or near "Health"
LINE 1: ...hern Ireland (under powers available through the 'Health and...
                                                             ^
SQL state: 42601
Character: 792
```

Comment (segfault) [reply migrations/202008131450.do.nys-settings.sql]: Fixing.

Comment (segfault) [reply migrations/202008131450.do.nys-settings.sql]: I don't think I need the languages quite like that based on what I'm seeing from Ireland's prod instance.

Comment (segfault) [reply migrations/202008131450.do.nys-settings.sql]: Updated and now matching more from prod.

**Review by dennisgove — APPROVED** [review]
lgtm - confirmed these query statements execute fine.

**Pull request merged**